cpp/LinksPSQL.h

```diff
@@ -0,0 +1,122 @@
+#include <pqxx/pqxx>
+
+namespace Platform::Data::Doublets
+{
+    template<typename TLink>
+    struct LinksPSQL: public ILinks<LinksPSQL<TLink>, TLink>

```

Comment:
Use guidelines formatting 
```suggestion
    struct LinksPSQL : public ILinks<LinksPSQL<TLink>, TLink>

```